### DISCUSSION TITLE
How to use correct Cron() function

### DISCUSSION THREAD

**Discussion by Dregorio1** [go-co-op/gocron#639] [Q&A] [answered]
Hello I have code like:
```
// gocron v1.35.2
 
jobs := []string{
"5 * * * ? *", //At 5 minutes past the hour
"30 * * * ? *", //At 30 minutes past the hour
}	
s := gocron.NewScheduler(time.UTC)
for _, j := range jobs {
	_, err := s.CronWithSeconds(j).SingletonMode().Do(doTask)
	if err != nil {
		log.Fatal(err)
	}
}
```
I use `CronWithSeconds()` but now I think that I use it wrong. I use this site https://crontab.cronhub.io/ to check the cron. I need parse `"5 * * * ? *" -> At 5 minutes past the hour`. How can I parse my string to get "proper" time? `CronWithSeconds()`  do not do what I want.

**Comment by JohnRoesler** [comment]
CronWithSeconds adds an additional field to the left:

So, take your cron tab for every 5 minutes `*/5 * * * *` and prefix with another * -> `* */5 * * * *`

**Reply by Dregorio1** [reply]
No, I want to get `At 5 minutes past the hour`. Maybe I wrongly describe my problem. I thought that `6*` has "one" standard. Like in AWS `6*`  for `"5 * * * ? *"` return `At 5 minutes past the hour` and that is what I want to achieve in my code. Also I used  https://crontab.cronhub.io/ this website to parse my `6*` cron `"5 * * * ? *"`, and I got what I wanted. So I thought `CronWithSeconds` also works this way. If not, what function should I use to achieve it?

**Reply by JohnRoesler** [reply, marked as answer]
Do you need seconds at all? If not just use `Cron()`. If you need to specify the seconds, then it's a standard crontab -> https://crontab.guru/ with the first field (on the left) being seconds

**Reply by Dregorio1** [reply]
I have `6*` and I cannot change that. This https://crontab.cronhub.io/ website parse it correctly (how I want). How can I achieve it in this library?

**Reply by Dregorio1** [reply]
> Do you need seconds at all? If not just use `Cron()`. If you need to specify the seconds, then it's a standard crontab -> https://crontab.guru/ with the first field (on the left) being seconds

Ok I got it. Thank you :)
